middleware/kubernetes/README.md

```diff
@@ -121,6 +121,42 @@ kubernetes coredns.local {
 	# Each line consists of the name of the federation, and the domain.
 	federation myfed foo.example.com
 	
+	# autopath [<host-domain>]
+	#
+	# Enables server side search path lookups for pods.  When enabled, coredns
+	# will identify search path queries from pods and perform the remaining path
+	# lookups on the pod's behalf.  The search path used mimics the resolve.conf
+	# search path deployed to pods. E.g.
+	#
+	#     search namespace.svc.cluster.local svc.cluster.local cluster.local
+	#
+	# <host-domain> is optional. If specified, coredns performs a lookup on this
```

Comment:
ok.  k8s allows > 1 host domain. they are supposedly copied from domain entries from the resolv.conf of the host node.